lib/Minz/Log.php

```diff
@@ -15,10 +15,10 @@ class Minz_Log {
 	 * NOTICE erreurs mineures ou messages d'informations
 	 * DEBUG Informations affichées pour le déboggage
 	 */
-	const ERROR = 2;
-	const WARNING = 4;
-	const NOTICE = 8;
-	const DEBUG = 16;
+	const ERROR = LOG_ERR;
```

Comment:
@marienfressinaud I suggest removing those custom constants and use PHP native ones instead https://php.net/function.syslog
What do you think?

Reply:
Merging to do my own final test before the call to testing, but feel free to reintroduce them if you miss them :-)